src/nvim/memline.c

```diff
@@ -1253,8 +1253,10 @@ void ml_recover(void)
       mf_put(mfp, hp, false, false);
     mf_close(mfp, false);           /* will also free(mfp->mf_fname) */
   }
-  free(buf->b_ml.ml_stack);
-  free(buf);
+  if (buf != NULL) { //may be NULL if swap file not found.
```

Comment:
Should be two spaces after `{`.

I think it might be a good idea to clean up files so we can take them off of `clint-ignored-files.txt`, but of course later.